README.md

```diff
@@ -10,6 +10,22 @@ $ yarn add -D sapui5-runtime
 ```
 This will download and unzip the latest SAPUI5 runtime.
 
+## Installing a specific Version
+In case you need a specific version of SAPUI5 you can specify it in your `package.json` before installing it.
+Add the following line to your `package.json`
+
+```JSON
+"sapui5RuntimeVersion": "X.Y.Z"
+``` 
+
+where `X.Y.Z` is a valid version from [https://tools.hana.ondemand.com/](https://tools.hana.ondemand.com/#sapui5)
+
+Afterwards proceed with the normal installation 
```

Comment:
Maybe we should change this line to: "Remember to run `npm install` or `yarn install` again after specifing your desired version."

This makes it a little more clear, that you need to run these command again, even if the runtime is already installed.